cpg_workflows/workflow.py

```diff
@@ -1239,6 +1254,42 @@ def queue_for_cohort(self, cohort: Cohort) -> dict[str, StageOutput | None]:
 
         return output_by_target
 
+    def queue_for_multicohort(self, multicohort: MultiCohort) -> dict[str, StageOutput | None]:
+        """
+        Plug the stage into the workflow.
+        """
+        output_by_target: dict[str, StageOutput | None] = dict()
+        for cohort in multicohort.get_cohorts():
+            for dataset in cohort.get_datasets():
```

Comment:
Do we need to emit a message for no datasets, like the previous function does?

Reply:
Added the warning. What is more important (that this comment highlighted to me) is we don't do a warning for no cohorts in the multi-cohort. Added that too!

Reply:
Although these warnings in general are less important now adays, historically and as per 'The Old Way' a user could specify 'skip_datasets' so you could find yourself in a position where all your specified datasets have been skipped -- hence the message. But we're scrapping this functionality, so arguably, we shouldn't encounter them.